doc/newsboat.asciidoc

```diff
@@ -76,15 +76,49 @@ Developers will also need:
 
 === Compiling and Installing
 
-After you've downloaded and installed the dependencies mentioned above, you can
-start compiling and installing newsboat. To compile newsboat, simply run
-`make` in the source tree. After a short time, this should complete
-successfully, and you can go on with installation by running `make install`. By
-default, this will install the `newsboat` binary to the _/usr/local/bin_
-directory. You can provide an alternative installation path using the `prefix`
-parameter, e.g. running `make install prefix=/opt/newsboat` will install the
-binary to the directory _/opt/newsboat/bin_.
+// UPDATE README.md IF YOU CHANGE THIS LIST
+
+There are numerous ways:
+
+- install from your distribution's repository
+  (https://repology.org/project/newsboat[a lot of distros have a package]);
```

Comment:
The installation chapter is now a bit confusing as it starts with "This chapter describes how to compile and install newsboat from source." and here goes on to explain how to install it from binary repositories. I'd split it up into "Installing" and "Building From Source" chapters.

Reply:
Thanks! I rearranged things; please take a look.

I kept README unchanged because we're using different wording there, so a single-list approach works fine there.

Reply:
Looks good to me :slightly_smiling_face: